### DISCUSSION TITLE
Change the way the desktop zoom shortcuts work to be better for people with visual impairments

### DISCUSSION THREAD

**Discussion by osdcdkje** [linuxmint/discussions#398] [Ideas]
I am a visually impaired person who has to use the desktop zoom feature almost all the time. To be able to zoom the desktop using keyboard shortcuts, people with visual impairments will first have to do the not-so-easy task of using their low-vision eyes to find the location where the zoom function is located and use the cursor to turn it on. On the login screen, I can't turn on the zoom function to check and correct my login information if something is incorrect. Users with visual impairments need to be able to enable zoom immediately with keyboard shortcuts both after and during login.

The default keyboard shortcuts "Super+Alt++" and "Super+Alt+-" are not convenient,. They should be changed to "Super++" and "Super+-". These shortcuts are shorter and can be pressed with one hand on a full-sized keyboard.

If I do not turn off the zoom function before shutting down or restarting, in the next session the desktop will be zoomed to the last level used. Pressing "Super+Alt++" will not increase the zoom level but return it to the default value. After changing the default shortcut to "Super++", pressing it should increase the zoom ratio.

By default, each time the desktop is zoomed in or out, the zoom ratio will increase or decrease by 0.2 times (for example, 1.0 - 1.2 - 1.4 - 1.6 - 1.4 - 1.2 - 1.0). Such increases and decreases are so small that they make little difference compared to the previous zoom level. It is increased by 1 time, meaning we will have increases and decreases like 1.0 - 2.0 - 3.0 - 2.0 - 1.0, etc.

After desktop zoom is enabled, a zoom bar should appear on the desktop. It will contain buttons that allow the user to zoom in or out on the desktop using the cursor, a field to type intermediate values ​​between zoom levels by default, such as 2.1, 2.5, 2.9, and a button to switch to the desktop zoom function settings window.

There should be an additional shortcut "Super+0" to quickly return the zoom level to the default value when needed. After magnifying the desktop up to nearly 10 times, pressing "Super+-" many times takes a bit of time.

There's something wrong with the way Linux Mint zooms, everything becomes blurry even at just 2x zoom. Developers need to make changes to make the image smoother and clearer.

**Comment by Jopp-gh** [comment]
Off topic, maybe useful for you (with best intentions):

I would check out the "bates method". Of course wikipedia and mainstream warns against this method because they are all pro school medicine which is a gigantic business model. When I worked at the red cross my superior did forbit me to give alternative and simple natural methods to help patients. 

But to get an idea, please check by yourself in what this bates method consists to get a picture what I'm talking about. You should find nicely illustrated pdf books online.
